packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -684,6 +687,14 @@ contract VotingV2 is
         return voteTiming.computeCurrentPhase(getCurrentTime());
     }
 
+    /**
+     * @notice Returns the phase length set in vote timing.
+     * @return uint256 the phase length.
+     */
+    function getPhaseLength() public view returns (uint256) {
+        return voteTiming.phaseLength;
+    }
+
```

Comment:
nit: any reason we can't just get this from the `voteTiming` public variable?

Reply:
it's true! we can. going to remove this. I did not think it worked but turns out we can get _both_ states within the lib really easily with this:
<img width="467" alt="CleanShot 2022-08-19 at 16 37 21@2x" src="https://user-images.githubusercontent.com/12886084/185643117-d9c40a48-6b39-485c-92be-c00036a51409.png">